README.md

```diff
@@ -18,6 +32,12 @@ Start the development server on http://localhost:3000
 yarn dev
 ```
 
+Start the backend development server on http://localhost:4000/graphiql
```

Comment:
How about setup indexer for Sigma Dao?

Reply:
We are not setting up indexer from Sigma Dao. We have removed the indexer client from sigma dao.

Reply:
No, I mean for deploy DAO approval app and setup db for graph ql.

Reply:
Yes, correct. We should add it. I have added it now. I would prefer link the `examples/dao` README. This would result in less maintenance of this README in case if examples/dao readme updates with steps, which is very much likely to happen.